example/glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -176,6 +188,23 @@ MM_CollectorLanguageInterfaceImpl::parallelDispatcher_handleMasterThread(OMR_VMT
 	/* Do nothing for now.  only required for SRT */
 }
 
+void
+MM_CollectorLanguageInterfaceImpl::generationalWriteBarrierStore(OMR_VMThread *omrThread, omrobjectptr_t parentObject, fomrobject_t *parentSlot, omrobjectptr_t childObject)
+{
+	GC_SlotObject slotObject(omrThread->_vm, parentSlot);
+	slotObject.writeReferenceToSlot(childObject);
+	MM_GCExtensionsBase *extensions = (MM_GCExtensionsBase *)omrThread->_vm->_gcOmrVMExtensions;
```

Comment:
There is an API on MM_GCExtensionsBase for getting the extensions from an OMR_VM.

MM_GCExtensionsBase::getExtensions(omrThread->_vm):